resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+                                   :value="setDefaultProject" required></typeahead>
+                    </div>
+
+                    <div class="checkbox">
+                        <label><input type="checkbox" id="files" v-model="cloneFiles">
+                            Filter <span>
+                            @if($volume->isImageVolume())
+                                    images
+                                @else
+                                    videos
+                                @endif
+                            </span>
+                        </label>
+                    </div>
+                    <div v-if="cloneFiles">
+                        <div id="file-panel" class="panel panel-default volume-files-panel">
+                            <div class="panel-heading">
+                                <div class="form-group">
+                                    <label>
+                                        @if($volume->isImageVolume())
+                                            Image(s):
+                                        @else
+                                            Video(s):
+                                        @endif
+                                    </label>
+                                    @if ($volume->isImageVolume())
+                                        <input type="text" class="form-control" id="files"
+                                               placeholder="img*.jpg" v-model="filePattern" required
+                                               v-on:keydown.enter="loadFilesMatchingPattern">
+                                    @else
+                                        <input type="text" class="form-control" id="files"
+                                               placeholder="video*.mp4" v-model="filePattern" required
+                                               v-on:keydown.enter="loadFilesMatchingPattern">
+                                    @endif
+                                </div>
+                            </div>
+                            <div class="panel-body">
+                                <ul class="list-group files-list" v-cloak>
+                                    <li v-for="file in selectedFiles" class="list-group-item"><span
+                                            class="text-muted">#<span v-text="file.id"></span></span> <span
+                                            v-text="file.filename"></span></li>
+                                </ul>
+                            </div>
+                            <div class="form-group{{ $errors->has('clone_files') ? ' has-error' : '' }}">
+                                @if($errors->has('clone_files'))
+                                    <span class="help-block">{{ $errors->first('clone_files') }}</span>
+                                @endif
+                            </div>
+                        </div>
+                    </div>
+                    <div v-cloak>
```

Comment:
You should not use v-cloak here because the checkboxes should always be visible. Instead, use `v-cloak` on the child elements that use `v-if` and should be hidden initially.